Mathlib/Topology/Clopen.lean

```diff
@@ -20,9 +20,7 @@ A clopen set is a set that is both open and closed.
 
 open Set Filter Topology TopologicalSpace Classical
 
-universe u v
-
-variable {α : Type u} {β : Type v} {ι : Type*} {π : ι → Type*}
+variable {α : Type*} {β : Type*} {ι : Type*} {π : ι → Type*}
```

Comment:
arguably a better choice for this type of thing is
```
universe uα uβ uι uπ
variable {α : Type uα} {β : Type uβ} {ι : Type uι} {π : ι → Type uπ}
```

This generates much nicer names than `Type*`, and as a result means that when faced with universe errors, it's much easier to tell what is going wrong.

`Type*` is a great compromise for when you're feeling lazy, but if files already go to the effort of naming universes manually, I'd argue that replacing them with `Type*` is a regression.

Reply:
Good point. I pushed a commit addressing this.